alphastats/plots/intensity_plot.py

```diff
@@ -52,12 +53,15 @@ def __init__(
         self.intensity_column = intensity_column
         self.preprocessing_info = preprocessing_info
 
-        self.protein_id = protein_id
+        self.protein_id = [protein_id] if isinstance(protein_id, str) else protein_id
         self.group = group
         self.subgroups = subgroups
         self.method = method
         self.add_significance = add_significance
         self.log_scale = log_scale
+        self.y_axis = self.intensity_column.replace("[sample]", "").strip()
+        if self.preprocessing_info[PreprocessingStateKeys.LOG2_TRANSFORMED]:
+            self.y_axis = "log2(" + self.yaxis + ")"
```

Comment:
this is used in l. 147 to do some df operations, meaning there is some implict coupling to another paet of the code (i.e. where the `log2(..)` is added to the column name) .. can this coupling be made explicit?

Reply:
The df operation is only to save all the lines of overwriting the label. The df is temporary. The typo is fixed in the next PR.